src/components/Landing.js

```diff
@@ -59,6 +70,36 @@ export default class Landing extends Component {
       document.title = `Pain Management Summary - ${patientName}`;
     }
   }
+  /*
+   * function for retrieving data from other sources e.g. PDMP
+   */
+  async getExternalData() {
+      const externalDatasetKey = 'ExternalDataSet';
+      let dataSet = {};
+      dataSet[externalDatasetKey] = {};
+      //PDMP data
+      //loading local file - this is temporary, just to demo rendering
+      //let response = await fetch(`${process.env.PUBLIC_URL}/pdmp.json`, {mode: 'no-cors'})
+      /*
+       * IF environmental variable REACT_APP_CONF_API_URL is set, call that endpoint
```

Comment:
Can you remove this code if you don't think you'll use it, or maybe make conditional on `REACT_APP_CONF_API_URL`?